pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -3032,6 +3032,13 @@ with haskellLib;
     })
   ] (doJailbreak super.http2-client);
 
+  # Needs tls >= 2.1.10
+  http2-tls = super.http2-tls.override { tls = self.tls_2_1_12; };
+
+  # Relax http2 version bound (5.3.9 -> 5.3.10)
+  # Tests fail with duplicate IsLabel instance error
+  grapesy = dontCheck (doJailbreak super.grapesy);
```

Comment:
Given that upstream has a very explicit bound for this, including a comment about tests... we should probably *not* just jailbreak and disable tests.

https://github.com/well-typed/grapesy/blob/4ce0c73ba1bb3c0d48efbed98a39201cd1ba2712/grapesy/grapesy.cabal#L177-L181

Can you create an upstream issue / PR first? Maybe see how to fix the tests?

Reply:
Yeah, a link to the upstream issue should then be added to the code comment.